### DISCUSSION TITLE
How To Get Sponsors?

### DISCUSSION THREAD

**Discussion by badcast** [community/community#64622] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How To Get Sponsors?

**Comment by arafajs** [comment, marked as answer]
Tips on how to get sponsors on GitHub:

- Build a great open source project. This is the most important thing you can do to attract sponsors. Your project should be well-maintained, well-documented, and solve a real problem.
- Have a strong online presence. This includes having a well-maintained GitHub profile, a blog, and social media presence. Be active in the open source community and contribute to other projects.
- Set clear sponsorship goals. What do you want to achieve with the sponsorship? Do you want to hire a new developer? Buy new hardware? Attend a conference? Be specific about your goals and how sponsors can help you achieve them.
- Offer different sponsorship tiers. This will allow sponsors to choose a level of support that is right for them. Some common sponsorship tiers include:

1. Bronze: $5/month
2. Silver: $10/month
3. Gold: $20/month
4. Platinum: $50/month

- Provide benefits to sponsors. This could include:

1. Early access to new features
2. Your name or logo on your project's website
3. A shout-out on your project's social media pages
4. A dedicated Slack channel for sponsors

- Be persistent. It takes time to attract sponsors. Don't give up if you don't get any sponsors right away. Keep building your project, growing your online presence, and reaching out to potential sponsors.
